.github/workflows/.github-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Node.js CI
+
+# Run on every push or pull request to the 'main' branch (you can change this to your preference)
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]  # You can adjust the Node.js version according to your project
```

Comment:
The `nvmrc` file points to node 20, should we also use that here for consistency?

Reply:
It's true, I change it!